misk/src/main/kotlin/misk/tasks/RepeatedTaskQueue.kt

```diff
@@ -68,10 +68,15 @@ class RepeatedTaskQueue @VisibleForTesting internal constructor(
     }, executor())
   }
 
+  override fun startUp() {
+    if (!running.compareAndSet(false, true)) return
```

Comment:
why have an if here?

Reply:
Good question - it was already there, I think it's only in the diff because of formatting.

Reply:
i imagine it was copied from the `triggerShutdown()` below, or there was previously a log/other code in this method - i'm pretty certain you could replace with `running.compareAndSet(false, true)` but if someone did add code below, they might not remember to put it back